src/mlmc/sample_storage_hdf.py

```diff
@@ -218,9 +220,7 @@ def save_n_ops(self, n_ops):
         :return: None
         """
         for level_id, (time, n_samples) in n_ops:
-            if n_samples == 0:
-                self._level_groups[level_id].n_ops_estimate = 0
-            else:
+            if n_samples > 0:
```

Comment:
Sure, this more correct. But what happens if there are no samples? How is n_ops_estimate initialized?